### ISSUE TITLE
Hide popup count from screen reader.

### ISSUE THREAD

**Issue by joe-allen-89** [cgkineo/adapt-hotgrid#111]
### Subject of the issue/enhancement/features
To be consistent with accessibility changes to the hotgraphic the popup count numbers in the popup toolbar should be hidden from the screen reader. 

### Your environment
* v5.25.2
* Chrome with NVDA & Jaws 2023

### Steps to reproduce
Using a screen reader open a hotgrid item and navigate through the popup toolbar
